<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>电商交易平台</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageLogin">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="4">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="2">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Agency FB</family>
              </font>
             </property>
             <property name="text">
              <string>账号：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="8">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="4">
            <widget class="QLineEdit" name="Password">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>密码：</string>
             </property>
            </widget>
           </item>
           <item row="4" column="8">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="Account">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="LoginButton">
             <property name="text">
              <string>登录</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>35</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="RegisterButton">
             <property name="text">
              <string>注册</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>43</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
          <zorder>RegisterButton</zorder>
          <zorder>LoginButton</zorder>
          <zorder>comboBox</zorder>
          <zorder>horizontalSpacer_8</zorder>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageManagement">
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1</y>
          <width>381</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addButton">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>增加</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteButton">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancelButton">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>撤销修改</string>
        </property>
       </widget>
       <widget class="QPushButton" name="submitButton">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>提交修改</string>
        </property>
       </widget>
       <widget class="QPushButton" name="discountButton">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>150</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>打折</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageInformation">
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>481</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>欢迎，</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>100</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Narrow</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuuser">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionlogin"/>
    <addaction name="separator"/>
    <addaction name="actionchange_password"/>
    <addaction name="separator"/>
    <addaction name="actionbalance_management"/>
    <addaction name="separator"/>
    <addaction name="actionlogout"/>
   </widget>
   <widget class="QMenu" name="menugoods">
    <property name="title">
     <string>商品</string>
    </property>
    <addaction name="actionmanagement"/>
    <addaction name="separator"/>
    <addaction name="actioninformation"/>
    <addaction name="separator"/>
    <addaction name="actionsearch"/>
   </widget>
   <addaction name="menuuser"/>
   <addaction name="menugoods"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionlogin">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Images/login.png</normaloff>:/Images/login.png</iconset>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="actionchange_password">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Images/password.png</normaloff>:/Images/password.png</iconset>
   </property>
   <property name="text">
    <string>密码设置</string>
   </property>
  </action>
  <action name="actionbalance_management">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Images/balance.png</normaloff>:/Images/balance.png</iconset>
   </property>
   <property name="text">
    <string>余额管理</string>
   </property>
  </action>
  <action name="actionlogout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Images/logout.png</normaloff>:/Images/logout.png</iconset>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="actioninformation">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Images/goods_information.png</normaloff>:/Images/goods_information.png</iconset>
   </property>
   <property name="text">
    <string>商品信息</string>
   </property>
  </action>
  <action name="actionsearch">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Images/search.png</normaloff>:/Images/search.png</iconset>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </action>
  <action name="actionmanagement">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Images/goods_management.png</normaloff>:/Images/goods_management.png</iconset>
   </property>
   <property name="text">
    <string>商品管理</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
